src/lab11.py

```diff
@@ -70,30 +70,73 @@ def parse_blend_mode(blend_mode_str):
         return skia.BlendMode.kMultiply
     elif blend_mode_str == "difference":
         return skia.BlendMode.kDifference
+    elif blend_mode_str == "destination-in":
+        return skia.BlendMode.kDstIn
     else:
         return skia.BlendMode.kSrcOver
 
 def linespace(font):
     metrics = font.getMetrics()
     return metrics.fDescent - metrics.fAscent
 
-class SaveLayer:
-    def __init__(self, sk_paint, children,
-            should_save=True, should_paint_cmds=True):
-        self.should_save = should_save
-        self.should_paint_cmds = should_paint_cmds
-        self.sk_paint = sk_paint
+class Alpha:
```

Comment:
I guess these classes are only there to make it easier to refer to them in the chapter. But it's dead code, which is something we don't have anywhere else...

Reply:
Yeah, I can put them in the example file for compare to do it’s thing.